packages/terra-base/tests/nightwatch/NoTranslationsLoadedBase.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Base from '../../lib/Base';
+
+const BaseNoTranslationsLoaded = () => {
+  const noTranslations = (
+    <Base locale="gabcdef" translationsLoadingPlaceholder={<div>No Translations</div>}>
```

Comment:
I'm going to keep it...but i might add a comment that this is ok to remove if it starts causing problems.